mmpose/codecs/mono_pose_lifting.py

```diff
@@ -0,0 +1,206 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+from copy import deepcopy
+from typing import Optional, Tuple
+
+import numpy as np
+
+from mmpose.registry import KEYPOINT_CODECS
+from .base import BaseKeypointCodec
+from .utils import camera_to_image_coord
+
+
+@KEYPOINT_CODECS.register_module()
+class MonoPoseLifting(BaseKeypointCodec):
```

Comment:
Compared with `ImagePoseLifting` and `VideoPoseLifting`, `MonoPoseLifting` is a little bit confusing.

Reply:
Agreed. This name is temporary because I didn't come up with a reasonable name.